src/library/scala/collection/immutable/HashSet.scala

```diff
@@ -988,7 +988,7 @@ private final class BitmapIndexedSetNode[A](
               val elementHash = improve(elementUnimprovedHash)
               subNode.contains(payload, elementUnimprovedHash, elementHash, shift + BitPartitionSize)
             }
-          } else {
+          } else ((node.dataMap & bitpos) == 0) && {
```

Comment:
One-element subnodes are eliminated it seems, so if `this` has a subnode at `bitpos` there are at least 2 elements, so there needs to be a subnode in `that`

Reply:
At first I wrote `else if (x != 0) false` which I could have commented appropriately, but then I was seduced by the boolean expression.